docs/partials/_constitution-content-partial.md

```diff
@@ -100,7 +100,7 @@
 <h3 id="section-4-security-council-elections">Section 4: Security Council Elections</h3>
 <p>The Security Council has 12 members, who are divided into a September Cohort of 6 members, and a March Cohort of 6 members. Every year on September 15, 12:00 UTC, an election starts for the 6 September Cohort seats; and every year on March 15, 12:00 UTC, an election starts for the 6 March Cohort seats.</p>
 <p>This means that the initial September Cohort will serve an initial term of 6 months, whereas the initial March Cohort will serve an initial term of 1 year.</p>
-<p>The initial Security Council Cohorts were determined by randomly splitting the 12 members into two 6-member cohorts - 6 members in the September Cohort and 6 members in the March Cohort. The members of the initial Security Council Cohorts are detailed in AIP-1.</p>
+<p>The initial Security Council Cohorts were determined by randomly splitting the 12 members into two 6-member cohorts - 6 members in the September Cohort and 6 members in the March Cohort. The members of the initial Security Council Cohorts are detailed in a transparency report <a href="https://docs.arbitrum.foundation/foundational-documents/transparency-report-initial-foundation-setup">here</a>.</p>
```

Comment:
would be cool if we provided an IPFS CID of the document instead of url since the url can be rugged and no integrity check on the document

Reply:
I like both (doc +. IPFS mirror link). The IPFS one shouldn't include the whole docs build (would include old version of constitution, other parts of the docs can / will change). So IPFS doc should either be raw/ugly version of transparency report, or if pretty / docusaurus markdown-rendered version, a version of these docs with all of the other content removed.

Reply:
did just IPFS, ugly version, to keep simple